style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  display: flex;
+
+  font-size: 30px;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background: #fbfbfb;
+  color: #cacaca;
```

Comment:
group similar properties !

Reply:
definately

Reply:
i have grouped the properties thanks for explaining me in meet